src/signers/RelaySigner.ts

```diff
@@ -0,0 +1,202 @@
+import { ethers } from 'ethers'
+
+import { TLProvider } from '../providers/TLProvider'
+import { TLSigner } from './TLSigner'
+
+import { Signature, UserObject } from '../typings'
+
+/**
+ * The RelaySigner class contains wallet related methods.
+ */
+export class RelaySigner implements TLSigner {
+  public provider: TLProvider
+
+  private wallet: ethers.Wallet
+
+  constructor(provider: TLProvider) {
+    this.provider = provider
+  }
+
+  ///////////////
+  // Accessors //
+  ///////////////
+
+  public get address(): string {
```

Comment:
We have now address and getAddress, I think we should only have one.

Reply:
Yes they are a bit redundant. I also wanted to only have `address` but `getAddres()` is a property from `ethers.Signer`. So maybe we should only keep `getAddress()`?